src/sandstorm/run-bundle.c++

```diff
@@ -1162,12 +1162,23 @@ private:
 
     // The environment inherited from the host is probably no good for us. E.g. an oddball
     // locale setting can crash Mongo because we don't have the appropriate locale files available.
+    //
+    // That said, there are a few environment variables that we do re-export.
+    char * http_proxy, * https_proxy;
```

Comment:
Oh, I was mirroring the environment variable name. +1, I can use camelCase for this.